src/NuGet.Core/NuGet.PackageManagement/Resolution/GatherExceptionHelpers.cs

```diff
@@ -43,6 +46,12 @@ public static class GatherExceptionHelpers
 
                 if (configEntry != null)
                 {
+                    logger.LogMinimal(string.Format(
```

Comment:
This is info message which should be logged with default logging level and LogMinimal logs all messages as Info only. LogInformation api logs it as Debug.